bluesky/plan_stubs.py

```diff
@@ -708,7 +708,7 @@ def open_run(md=None):
 
     See Also
     --------
-    :func:`bluesky.plans.close_run`
+    func:`bluesky.plans_stubs.close_run`
```

Comment:
A leading colon was lost here. The `:func:` is sphinx code for, "Try to find a function with this name and turn this into a link to its API documentation."